service/create/loadbalancer.go

```diff
@@ -65,18 +67,31 @@ func (s *Service) createLoadBalancer(input LoadBalancerInput) error {
 		s.logger.Log("debug", fmt.Sprintf("hosted zone '%s' already exists, reusing", hz.Name))
 	}
 
-	recordSet := &awsresources.RecordSet{
+	apiRecordSet := &awsresources.RecordSet{
 		Client:       input.Clients.Route53,
 		Resource:     lb,
 		Domain:       input.Cluster.Spec.Cluster.Kubernetes.API.Domain,
 		HostedZoneID: hz.ID(),
 	}
 
-	if err := recordSet.CreateOrFail(); err != nil {
+	if err := apiRecordSet.CreateOrFail(); err != nil {
 		return microerror.MaskAny(fmt.Errorf("error registering DNS '%s'", errgo.Details(err)))
 	}
 
-	s.logger.Log("debug", fmt.Sprintf("created or reused DNS record for ELB"))
+	s.logger.Log("debug", fmt.Sprintf("created or reused DNS record for api"))
+
+	etcdRecordSet := &awsresources.RecordSet{
+		Client:       input.Clients.Route53,
+		Resource:     lb,
+		Domain:       input.Cluster.Spec.Cluster.Etcd.Domain,
+		HostedZoneID: hz.ID(),
+	}
+
+	if err := etcdRecordSet.CreateOrFail(); err != nil {
+		return microerror.MaskAny(fmt.Errorf("error registering DNS '%s'", errgo.Details(err)))
+	}
+
+	s.logger.Log("debug", fmt.Sprintf("created or reused DNS record for etcd"))
```

Comment:
When there is no variable to substitute there is no reason to use `fmt`. The same applies to the logging below.